ReactAndroid/src/main/java/com/facebook/react/shell/MainPackageConfig.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.react.shell;
+
+import com.facebook.imagepipeline.core.ImagePipelineConfig;
```

Comment:
There seems to be a problem with this import in CircleCI.

`com.facebook.imagepipeline.core.ImagePipelineConfig` does not exist.